scripts/update-version.py

```diff
@@ -0,0 +1,356 @@
+#!/usr/bin/python3
+# This script will to help to manage rhdm components modules version, it will update all needed files
+# Example of usage:
+#   # move the current version to the next one or rcX
+#   python scripts/update-version.py -v 7.15.1
```

Comment:
change to python3? and comment that is a dry-run and user would need to add `--confirm` flag to actually change the files

Reply:
surething, updated.